# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/Loader.h
# Commit: 4778a482abd6
# Full Hash: 4778a482abd6ba484aa2d8ec1062700bbfbd3062
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2024-10-02 09:50:09
# Regressor Bug: 1899734
# File Overlap Count: 2
# Description:
#   Bug 1899734 - Part 8: Add performance entry when stylesheet is created from a cache. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D223354
# ==============================================================================

diff -r b59dcea26b32 -r 4778a482abd6 layout/style/Loader.h
--- a/layout/style/Loader.h	Tue Oct 01 12:17:23 2024 +0000
+++ b/layout/style/Loader.h	Tue Oct 01 12:17:24 2024 +0000
@@ -584,6 +584,9 @@
   // Notify observers of a cached stylesheet being.
   void NotifyObserversForCachedSheet(SheetLoadData&);
 
+  // Add the Performance API's Resource Timing entry for the cached load data.
+  void AddPerformanceEntryForCachedSheet(SheetLoadData&);
+
   // Start the loads of all the sheets in mPendingDatas
   void StartDeferredLoads();
 